pkg/tfgen/test_data/signalfx-log-timeline/expected.json

```diff
@@ -1,5 +1,5 @@
 {
-  "Description": "You can add logs data to your Observability Cloud dashboards without turning your logs into metrics first.\n\nA log timeline chart displays timeline visualization in a dashboard and shows you in detail what is happening and why.\n\n## Example\n\n```tf\nresource \"signalfx_log_timeline\" \"my_log_timeline\" {\n  name        = \"Sample Log Timeline\"\n  description = \"Lorem ipsum dolor sit amet, laudem tibique iracundia at mea. Nam posse dolores ex, nec cu adhuc putent honestatis\"\n\n  program_text = <<-EOF\n  logs(filter=field('message') == 'Transaction processed' and field('service.name') == 'paymentservice').publish()\n  EOF\n\n  time_range = 900\n\n}\n```",
+  "Description": "You can add logs data to your Observability Cloud dashboards without turning your logs into metrics first.\n\nA log timeline chart displays timeline visualization in a dashboard and shows you in detail what is happening and why.\n\n## Example\n\n```tf\nresource \"signalfx_log_timeline\" \"my_log_timeline\" {\n  name        = \"Sample Log Timeline\"\n  description = \"Lorem ipsum dolor sit amet, laudem tibique iracundia at mea. Nam posse dolores ex, nec cu adhuc putent honestatis\"\n\n  program_text = \u003c\u003c-EOF\n  logs(filter=field('message') == 'Transaction processed' and field('service.name') == 'paymentservice').publish()\n  EOF\n\n  time_range = 900\n\n}\n```",
```

Comment:
This was unrelated to my change, and serializes the same (so tests pass with both representations).